### ISSUE TITLE
Use angular templating over jquery

### ISSUE THREAD

**Issue by estellecomment** [medic/medic-webapp#2188]
Angular has its own way of doing UIs : set vars in a controller, that change the corresponding view.
Using Jquery tends to ad messiness.

**Comment by alxndrsn** [comment]
Is this issue relating to any particular bit of code, or is it just general guidance?

**Comment by estellecomment** [comment]
General guidance. 
Overall, our app is angular, so does much of the styling and animations with templates, ng-if, ng-class, angular directives, all that.
If you’re using jquery, either you're using it in the controller that corresponds to the template, in which case it would be clearer to use the template (otherwise you have half your animations/styling in the template, and half in the controller)
Or you're using jquery someplace else, in which case you're editing random templates from random places and that's hard to follow when reading code, and hard to maintain.

I don't mean that we should never-ever use jquery ever. But we should prune down our use of it when it's peppered in the middle of angular.

**Comment by garethbowen** [comment]
Use angular is our policy. Where possible remove jquery in favour of angular as you find it. Closing this as it has no specific deliverable.
